Planter.sh fails

Following the instructions on the [planter.sh readme](https://github.com/kubernetes/test-infra/blob/master/planter/README.md):

```
To build kubernetes set up your $GOPATH/src to contain:

$GOPATH/src/k8s.io/kubernetes/   # ... <kubernetes/kubernetes checkout>
$GOPATH/src/k8s.io/test-infra/   # ... <kubernetes/test-infra checkout>
Then from $GOPATH/src/k8s.io/kubernetes/ run: ./../planter/planter.sh make bazel-build
```

produces:

```console
→ ./../test-infra/planter/planter.sh make bazel-build
0.18.0: Pulling from k8s-testimages/planter
Digest: sha256:3878e42a8bfc07ec34ca3905ee6c7418b44a36fcda9f6b47c9eec7f35d7c5c7b
Status: Image is up to date for gcr.io/k8s-testimages/planter:0.18.0
Starting local Bazel server and connecting to it...
ERROR: infinite symlink expansion detected
[start of symlink chain]
/Users/mmatthias/go/src/k8s.io/kubernetes
/Users/mmatthias/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes
[end of symlink chain]
ERROR: Failed to get information about path, for _output/local/go/src/k8s.io/kubernetes, skipping: Infinite symlink expansion
ERROR: Infinite symlink expansion
INFO: Elapsed time: 35.636s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (3382 packages loaded)
    currently loading: vendor/golang.org/x/text/cases ... (10 packages)
Makefile:556: recipe for target 'bazel-build' failed
make: *** [bazel-build] Error 1
```

I expected this command to run planter in a docker container, not produce an error.

--- a/planter/README.md
+++ b/planter/README.md
@@ -16,6 +16,9 @@ $GOPATH/src/k8s.io/test-infra/   # ... <kubernetes/test-infra checkout>
 Then from `$GOPATH/src/k8s.io/kubernetes/` run:
  `./../planter/planter.sh make bazel-build`.
 
+NOTE: if you previously built Kubernetes by other means, you may need to run
+`make clean` first to clean up some symlink cycles.
+
  For `test-infra` you can run eg `./planter/planter.sh bazel test //...`.
 
 ## Options

